changelogs/fragments/netcommon_httpapi_ciphers.yaml

```diff
@@ -0,0 +1,3 @@
+---
+minor_changes:
+  - Add optional netcommon_httpapi_ciphers for httpapi connection plugins to allow overriding default SSL/TLS ciphers.
```

Comment:
```suggestion
  - httpapi - Add option netcommon_httpapi_ciphers to allow overriding default SSL/TLS ciphers. (https://github.com/ansible-collections/ansible.netcommon/pull/494)
```

Reply:
Updated in https://github.com/ansible-collections/ansible.netcommon/pull/494/commits/73e9010179949b941481b40fcc9af1af91a5b0d3